lib/beautify-stack.js

```diff
@@ -1,13 +1,19 @@
 'use strict';
 
-function beautifyStack(stack) {
-	var re = /(?:^(?! {4}at\b).{6})|(?:\((?:[A-Z]:)?(?:[\\\/](?:(?!node_modules[\\\/]ava[\\\/])[^:\\\/])+)+:\d+:\d+\))/;
-	var found = false;
+var StackUtils = require('stack-utils');
+
+var stackUtils = new StackUtils({
+	internals: StackUtils.nodeInternals().concat([
+		/\(internal\/[\w_-]+\.js:[0-9]+:[0-9]+\)$/,
+		/\(native\)$/,
+		/[\\\/]ava[\\\/](?:lib[\\\/])?[\w-]+\.js:[0-9]+:[0-9]+\)?$/,
+		/[\\\/]node_modules[\\\/](?:bluebird|empower-core)[\\\/]/
+	])
+});
 
-	return stack.split('\n').filter(function (line) {
-		var relevant = re.test(line);
-		found = found || relevant;
-		return !found || relevant;
+function beautifyStack(stack) {
+	return stack.split('\n')[0] + '\n' + stackUtils.clean(stack).split('\n').map(function (s) {
```

Comment:
Yep. Whitespace at the beginning of any multi-line string makes the yaml output much harder to read. `stack-utils` trims whitespace from the first line.

I guess we could add an `indent` option to `stack-utils`.